contrib/runners/action_chain_runner/action_chain_runner/action_chain_runner.py

```diff
@@ -48,7 +48,7 @@
 from st2common.util import jinja as jinja_utils
 from st2common.util import param as param_utils
 from st2common.util.config_loader import get_config
-from st2common.util.ujson import fast_deepcopy
+from st2common.util.deep_copy import fast_deepcopy
```

Comment:
Can we call this module json util or something so not to be confused with copy.deepcopy?

Reply:
I intentionally renamed it to ``deep_copy`` since I think previous name was a bad one (I know, I picked it originally :D) - previous name was leaking implementation details which should not matter to the end user.

I think ``deep_copy`` is a better name since it conveys what the module is used for - deep copying dictionaries.

And the module is not JSON specific either, it's can deep copy arbitrary dictionaries with simple types. ``orjson`` is just an implementation detail of that function.

Reply:
I don't want this to be confused with copy.deepcopy which is not json or dict specific.

Reply:
Maybe I can rename it to ``fast_deepcopy_dict`` then?

Reply:
And I guess I need to be more explicitly - it doesn't just support dicts either (that's just how we use it).

It supports any kind of value as long as it only contains simple types (so no class instances, etc.).

Reply:
Per discussion on Slack, I pushed a couple of changes - add some more tests, update function docstring comment, rename it to ``fast_deepcopy_dict``.

As discussed, it can also be used on other simple values (think lists) and not just dictionaries, but using it on dictionaries with simple value types is our primary use case for it so I think that name is an OK compromise for now.